tests/parse/main.cpp

```diff
@@ -162,6 +174,10 @@ int main(int argc, char** argv, char** envp) {
         EXPECT(barrelRoll, true);
         EXPECT(flagsFound, std::string{"abc"});
 
+        EXPECT(categoryKeywordActualValues[0], "we are having fun");
```

Comment:
you should use .at(). If for whatever reason the handler fails, `[1]` will produce an access out-of-bounds, while .at will throw.